react-array-methods/src/components/Counter.tsx

```diff
@@ -1,7 +1,8 @@
-import React from 'react';
 import styled from 'styled-components';
 
 import { useSelector } from 'react-redux';
+import IState from '../store/IState';
+import ICounter from '../store/ICounter';
```

Comment:
I prefer adding Props or Types at end of named TS decr.
Should it be more consistent for whole project?

Reply:
Not sure I understand your comment... Can you please elaborate?